--- a/conf/conf.inc.php
+++ b/conf/conf.inc.php
@@ -145,3 +145,3 @@

-    define('P2_CONFIG_DIR', __DIR__);
+    define('P2_CONFIG_DIR', '/var/www/conf');

--- a/conf/conf_user_def.inc.php
+++ b/conf/conf_user_def.inc.php
@@ -537,10 +537,10 @@
 // プロキシを利用(する:1, しない:0)
-$conf_user_def['proxy_use'] = 0; // (0)
+$conf_user_def['proxy_use'] = 1; // (0)
 $conf_user_rad['proxy_use'] = array('1' => 'する', '0' => 'しない');

 // プロキシホスト ex)"127.0.0.1", "www.p2proxy.com"
-$conf_user_def['proxy_host'] = ""; // ("")
+$conf_user_def['proxy_host'] = "127.0.0.1"; // ("")

 // プロキシポート ex)"8080"
-$conf_user_def['proxy_port'] = ""; // ("")
+$conf_user_def['proxy_port'] = "9080"; // ("")

--- a/conf/conf_admin_ex.inc.php
+++ b/conf/conf_admin_ex.inc.php
@@ -103,3 +103,3 @@
 // ImageCache2（off:0, PCのみ:1, 携帯のみ:2, 両方:3）
-$_conf['expack.ic2.enabled'] = 0; // (0)
+$_conf['expack.ic2.enabled'] = 3; // (0)

--- a/conf/conf_ic2.inc.php
+++ b/conf/conf_ic2.inc.php
@@ -20,3 +20,3 @@
 // 注2: MySQL,PosrgreSQLでは予めデータベースを作っておく。
-$_conf['expack.ic2.general.dsn'] = "";
+$_conf['expack.ic2.general.dsn'] = 'sqlite:' . $_conf['db_dir'] . '/imgcache.sqlite';

# Via: https://pastebin.com/bVdiEsms
--- a/rep2/post.php
+++ b/rep2/post.php
@@ -143,7 +143,7 @@ if (!empty($_POST['newthread'])) {
 if (!empty($_POST['savedraft'])) {
     // 書き込みを一時的に保存
     $post_backup_key = PostDataStore::getKeyForBackup($host, $bbs, $key, !empty($_REQUEST['newthread']));
-    PostDataStore::set($post_backup_key, $post_cache);
+    PostDataStore::set($post_backup_key, mb_convert_encoding($post_cache, 'CP932', 'UTF-8,CP932,CP51932'));
     showPostMsg(false, '下書きを保存しました', false); // 書き込みが完了(第一引数がtrue)しないとリロードが効かないのでfalseに変更
     exit;
 }
@@ -209,7 +209,7 @@ PostDataStore::set($post_config_key, array(
 //================================================================

 // 書き込みを一時的に保存
-PostDataStore::set($post_backup_key, $post_cache);
+PostDataStore::set($post_backup_key, mb_convert_encoding($post_cache, 'CP932', 'UTF-8,CP932,CP51932'));

 // cookie 読み込み
 $cookie_key = $_login->user_u . '/' . P2Util::normalizeHostName(P2HostMgr::isHostBbsPink($host) ? 'www.bbspink.com' : (P2HostMgr::isHost2chs($host) ? 'www.5ch.net' : $host)); // 忍法帳対応

--- a/rep2/post_draft.php
+++ b/rep2/post_draft.php
@@ -33,7 +33,7 @@ header('Content-Type: text/plain; charset=UTF-8');
 $post_param_keys    = array('bbs', 'key', 'time', 'FROM', 'mail', 'MESSAGE', 'subject', 'submit');
 $post_internal_keys = array('host', 'sub', 'popup', 'rescount', 'ttitle_en');
 foreach ($post_param_keys as $pk) {
-    ${$pk} = (isset($_POST[$pk])) ? mb_convert_encoding($_POST[$pk], 'CP932', 'UTF-8') : '';
+    ${$pk} = (isset($_POST[$pk])) ? mb_convert_encoding($_POST[$pk], 'CP932', 'UTF-8,CP932,CP51932') : '';
 }
 foreach ($post_internal_keys as $pk) {
     ${$pk} = (isset($_POST[$pk])) ? $_POST[$pk] : '';
